src/components/overlays/SavePrompt.tsx

```diff
@@ -0,0 +1,66 @@
+import classnames from 'classnames';
+import _ from 'lodash';
+import React from 'react';
+import { FiArrowRight } from 'react-icons/fi';
+import { getLabel } from '~/codeLists/labelCodes';
+import * as s from './savePrompt.scss';
+
+const SavePrompt = (props: any) => {
```

Comment:
Always try to avoid using any. Especially here, now we lose the benefits of typescript